autoload/sneak/util.vim

```diff
@@ -120,6 +121,7 @@ endfunc
 
 func! s:init_hl() abort
   exec "highlight default Sneak guifg=white guibg=magenta ctermfg=white ctermbg=".(&t_Co < 256 ? "magenta" : "201")
+  exec "highlight default SneakCurrent guifg=white guibg=magenta ctermfg=white ctermbg=".(&t_Co < 256 ? "magenta" : "201")
```

Comment:
I'm not sure of the default color, put it same as the `Sneak` default for starters.

Reply:
Need a different color than `Sneak`. The current behavior intentionally does NOT highlight cur match, so changing it to be the same as `Sneak` completely reverses that.

either LightMagenta or DarkMagenta seems best. 

    hi Sneak guibg=LightMagenta ctermbg=LightMagenta guifg=0 ctermfg=0